server/index.js

```diff
@@ -1,35 +1,27 @@
 require('habitat').load();
 
-var Hapi = require('hapi');
-var locales = require('./locales');
-var reactRouted = require('../dist/lib/react-server-route.js')(locales);
-var getServerOptions = require('./get-server-options');
-var baseRoutes = require('./base-routes');
-var services = require('./services');
-var finalizeServer = require('./finalise-server');
+const Hapi = require('hapi');
+const AuthBearer = require('hapi-auth-bearer-token');
+const locales = require('./locales');
+const reactRouted = require('../dist/lib/react-server-route.js')(locales);
+const getServerOptions = require('./get-server-options');
+const baseRoutes = require('./base-routes');
+const services = require('./services');
+const finalizeServer = require('./finalise-server');
 
 const ONE_HOUR_MS = 1000 * 60 * 60;
 
+module.exports = async function(options) {
+  const serverOptions = getServerOptions(options);
+  const server = new Hapi.Server(serverOptions);
 
-module.exports = function(options) {
-  var serverOptions = getServerOptions(options);
-  var server = new Hapi.Server(serverOptions);
-
-  server.connection({
-    host: process.env.HOST,
-    port: process.env.PORT,
-    uri: process.env.APPLICATION_URI
-  });
-
-  server.register(require("hapi-auth-bearer-token"), function(err) {
-    if (err) {
```

Comment:
This error check serving no purpose?

Reply:
register no longer has a callback api and throws errors. they'll be caught in the Try/Catch wrapping any register calls we have

Reply:
k